README.md

```diff
@@ -1,38 +1,74 @@
----
-tags: ternary, ||=, operators
-language: ruby
-resources: 4
----
-
 # Operations
 
 ## Objectives
 
 Get familiar with: 
 
+* Learn about the ternary operator
+* Review using comparison operators and boolean operators
 * passing arguments to methods
```

Comment:
hmmm the point of a lab on operators also has the objective of passing arguments to methods?